src/firewall/core/fw.py

```diff
@@ -109,8 +109,9 @@ def __repr__(self):
             (self.__class__, self.ip4tables_enabled, self.ip6tables_enabled,
              self.ebtables_enabled, self._state, self._panic,
              self._default_zone, self._module_refcount, self._marks,
-             self.cleanup_on_exit, self.ipv6_rpfilter_enabled,
-             self.ipset_enabled, self._individual_calls, self._log_denied)
+             self.cleanup_on_exit, self.cleanup_modules_on_exit,
```

Comment:
You added `cleanup_modules_on_exit`, but forgot to add `%r` to the format string.